lib/models/storage_response_model.dart

```diff
@@ -0,0 +1,102 @@
+/// ok : true
+/// value : {"cid":"bafkreihlusofe353lfwntp7xcwvpzjvqzsohb7iykni3z7525gidsg3wqa","created":"2021-11-08T13:34:51.769+00:00","type":"image/jpeg","scope":"Easel_test","files":[],"size":101453,"pin":{"cid":"bafkreihlusofe353lfwntp7xcwvpzjvqzsohb7iykni3z7525gidsg3wqa","created":"2021-11-08T13:34:51.769+00:00","size":101453,"status":"queued"},"deals":[]}
+
+class StorageResponseModel {
+  bool? ok;
+  Value? value;
+
+  StorageResponseModel({this.ok, this.value});
+
+  StorageResponseModel.fromJson(dynamic json) {
+    ok = json['ok'];
+    value = json['value'] != null ? Value.fromJson(json['value']) : null;
+  }
+
+  Map<String, dynamic> toJson() {
+    var map = <String, dynamic>{};
+    map['ok'] = ok;
+    if (value != null) {
+      map['value'] = value?.toJson();
+    }
+    return map;
+  }
+}
+
+/// cid : "bafkreihlusofe353lfwntp7xcwvpzjvqzsohb7iykni3z7525gidsg3wqa"
```

Comment:
Is this some test data that's commented?

Reply:
It's there to help understand how the json response data looks like.

Reply:
These kind of json data can be in a file, in a design document. It has no place in the actual code as a comment. 
Commented code shouldn't be in the PRs. Comments should help:
1- Make the code more readable
2- Help build out documentation.